apps/fluent-tester/src/testPages.ts

```diff
@@ -95,11 +94,6 @@ export const tests: TestDescription[] = [
     component: TextTest,
     testPage: HOMEPAGE_TEXT_BUTTON,
   },
-  {
```

Comment:
I don't see the experimetnal text test page contents moving anywhere, so is removing the experimental test page from the list intended?

Reply:
Ah I was going to but I decided to keep the test pages as they are. I'll put them back in!